roles/nuage_master/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Install Nuage Kubemon
+  sudo: true
+  yum: name={{ nuage_kubemon_rpm }} state=present
+
+- name: set kubeconfig 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^kubeconfig line='kubeconfig{{':'}} {{ admin_config }}'
+
+- name: set openshift admin 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^openshiftAdmin line='openshiftAdmin{{':'}} {{ os_admin }}'
+
+- name: Set Openshift master config path
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^openshiftMasterConfig line='openshiftMasterConfig{{':'}} {{ master_config_yaml }}'
```

Comment:
@detiber thanks - do we expect users to _only_ use ansible to "update" parameters in the master-config? If not, it begs the question as to what hooks are available for plugins to re-read this information?